src/trusted/service_runtime/nacl_syscall_common.h

```diff
@@ -105,21 +110,21 @@ int32_t NaClSysStat(struct NaClAppThread *natp,
                     struct nacl_abi_stat *nasp);
 
 int32_t NaClSysMkdir(struct NaClAppThread *natp,
-                     uint32_t             path,
+                     uint32_t             pathname,
```

Comment:
Does that variable name make sense? This isn't a pathname in the common meaning of the word.

Reply:
This was to keep consistency with  the other system call parameter naming conventions.